slider/mod.py

```diff
@@ -147,9 +147,29 @@ def circle_radius(cs):
     return (512 / 16) * (1 - 0.7 * (cs - 5) / 5)
 
 
+class HitWindows(namedtuple('HitWindows', 'perfect, good, bad')):
+    """Times to hit an object at various accuracies
+
+    Parameters
+    ----------
+    perfect : int
```

Comment:
are these names used elsewhere in osu!? ideally, we would use `300`, `100`, `50`, but those are not identifiers. nametuple doesn't allow names like `_300`, `_100`, or `_50`, but maybe we could return a mapping structure where you could do:

```python
result[300]
result[100]
result[50]
```

idk if this is actually better though, what do you think?